node/src/types/status_feed.rs

```diff
@@ -167,16 +175,17 @@ impl From<Block> for MinimalBlockInfo {
 #[derive(PartialEq, Eq, Serialize, Deserialize, Debug, JsonSchema)]
 #[serde(deny_unknown_fields)]
 pub struct GetStatusResult {
+    /// The node ID and network address of each connected peer.
```

Comment:
Do such changes really matter? Afaik, order of fields in JSON doesn't matter. If we do it for convenience (human-readibility), are we sure that Serde gives us guarantee about order of the serialized fields?

From what I see, this kinda works in REST, but RPC API still returns fields in alphabetical order:

```
{
  "jsonrpc": "2.0",
  "id": "1",
  "result":{
    "api_version": "1.0.0",
    "available_block_range":{
      "high": 0,
      "low": 0
    },
    "block_sync":[],
    "build_version": "1.4.8-1ea1bdc@DEBUG",
     ...and so on...
```

Reply:
Imho, the change we made doesn't guarantee how the final JSON is rendered, but that's not a blocker.